ArduPlane/Attitude.cpp

```diff
@@ -501,7 +501,7 @@ void Plane::stabilize()
             plane.stabilize_pitch(speed_scaler);
         }
 #endif
-#if ENABLE_SCRIPTING
+#ifdef ENABLE_SCRIPTING
```

Comment:
Could we set `ENABLE_SCRIPTING` to zero in waf instead?

Reply:
only if you also change the 62 instances of "#ifdef ENABLE_SCRIPTING" to "#if"